examples/02_decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,87 @@
+"""
+Decoding with :class:`nilearn.decoding.Decoder`: face vs house recognition
+===========================================================================
+Here is a simple example of decoding with the decoder meta-estimator,
+reproducing a face vs house discrimination task on the study Haxby 2001.
+
+    * J.V. Haxby et al. "Distributed and Overlapping Representations of Faces
+      and Objects in Ventral Temporal Cortex", Science vol 293 (2001),
+      p 2425.-2430.
+
+"""
+######################################################################
+# Loading the data
+# -----------------
+
+# First, we load the Haxby dataset
+from nilearn.datasets import fetch_haxby
+data_files = fetch_haxby()
+
+func_filenames = data_files.func[0]
+labels_filenames = data_files.session_target[0]
+
+# print basic information on the dataset
+print('First subject functional nifti images (4D) are at: %s' %
+      func_filenames)  # 4D data
+
+# Load Target labels
+import numpy as np
+labels = np.recfromcsv(labels_filenames, delimiter=" ")
```

Comment:
We'll have to switch the example to use pandas, but let's do it after the merge.

Reply:
I can file an issue + PR for this but what is the rationale in general?